base/server/python/pki/server/__init__.py

```diff
@@ -1250,6 +1253,17 @@ def get_password(self, name):
         if name in self.passwords:
             return self.passwords[name]
 
+        # find password in keyring
```

Comment:
I see the code changes in here only add code to check keyring.  So this file originally only checks password.conf?

Reply:
yes. Originally, it checked password.conf. If not found, it prompted the user. This was the major cause for the nuxwdog to be broken in RHEL8. 

The original nuxwdog provides JNI. There is no python interface to actually query the nuxwdog server.

This new change will query the keyring directly.